package com.desafioitau.api.transferencia.v1.conta.exception;

public class ContaServiceUnavailableException extends ContaException {
    public ContaServiceUnavailableException(Throwable cause) {
        super(cause);
    }
}
